GameServerLib/GameObjects/AttackableUnits/AttackableUnit.cs

```diff
@@ -113,7 +116,7 @@ public class AttackableUnit : GameObject, IAttackableUnit
             Model = model;
             Waypoints = new List<Vector2> { Position };
             CurrentWaypoint = new KeyValuePair<int, Vector2>(1, Position);
-            CrowdControls = new List<ICrowdControl>();
+            Status = StatusFlags.CanAttack | StatusFlags.CanCast | StatusFlags.CanMove | StatusFlags.CanMoveEver;
```

Comment:
We have both function `CanAttack` and status flags with the same name, what is the reason they don't act the same?

Reply:
Usually Status is only changed by scripts. There can be cases where we have the CanAttack status, but we also have the Stunned status, which should still prevent us from attacking.